packages/core/test/unit/query-generator/remove-column-query.test.ts

```diff
@@ -1,63 +1,75 @@
-import { DataTypes } from '@sequelize/core';
 import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
 import { createSequelizeInstance, expectsql, getTestDialect, sequelize } from '../../support';
 
+const dialect = sequelize.dialect;
 const dialectName = getTestDialect();
 
+// TODO: add additional tests and type altered behaviour for sqlite
+
 describe('QueryGenerator#removeColumnQuery', () => {
   const queryGenerator = sequelize.getQueryInterface().queryGenerator;
 
-  const User = sequelize.define('User', {
-    firstName: DataTypes.STRING,
-    age: DataTypes.INTEGER,
-  }, { timestamps: false });
-
-  it('generates a DROP COLUMN query in supported dialects', () => {
-    expectsql(() => queryGenerator.removeColumnQuery(User.table, 'age'), {
-      default: `ALTER TABLE [Users] DROP COLUMN [age];`,
-      postgres: `ALTER TABLE "Users" DROP COLUMN "age";`,
-      snowflake: `ALTER TABLE "Users" DROP "age";`,
-      sqlite: 'CREATE TABLE IF NOT EXISTS `Users_backup` (`0` a, `1` g, `2` e);INSERT INTO `Users_backup` SELECT `0`, `1`, `2` FROM `Users`;DROP TABLE `Users`;ALTER TABLE `Users_backup` RENAME TO `Users`;',
-      'mariadb mysql': 'ALTER TABLE `Users` DROP `age`;',
+  it('generates a query that drops a column', () => {
+    expectsql(() => queryGenerator.removeColumnQuery('myTable', 'myColumn'), {
+      default: 'ALTER TABLE [myTable] DROP COLUMN [myColumn];',
+      'mariadb mysql snowflake': 'ALTER TABLE [myTable] DROP [myColumn];',
+      sqlite: 'CREATE TABLE IF NOT EXISTS `myTable_backup` (`0` m, `1` y, `2` C, `3` o, `4` l, `5` u, `6` m, `7` n);INSERT INTO `myTable_backup` SELECT `0`, `1`, `2`, `3`, `4`, `5`, `6`, `7` FROM `myTable`;DROP TABLE `myTable`;ALTER TABLE `myTable_backup` RENAME TO `myTable`;',
```

Comment:
Ooof, It's pretty bad that we accepted a PR that added a test like this one. In the future, sqlite should probably _not_ implement `removeColumnQuery` and instead provide a separate `replaceTableQuery`, considering this query works completely differently and has completely different requirements

Could you add a TODO for this?

Reply:
Added a TODO in the SqliteQueryGenerator and removed the TODO at the top of this file